src/main/resources/META-INF/mods.toml

```diff
@@ -38,7 +38,7 @@ MineColonies is a Colony/Town Simulator that adds many structures and NPC worker
     # Does this dependency have to exist - if not, ordering below must be specified
     mandatory=true #mandatory
     # The version range of the dependency
-    versionRange="[31,)" #mandatory
+    versionRange="[31.1.0,)" #mandatory
     # An ordering relationship for the dependency - BEFORE or AFTER required if the relationship is not mandatory
```

Comment:
this requires 31.1.0 and not 31.1, afaik

Reply:
[x,) stays for x and anything higher
basically x <= version

Reply:
ah okay,